.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -11,20 +11,20 @@ Please fill out the following template so we can reproduce and fix your issue as
 
 **Environment:**
 
-- Elements version:
+-   Elements version:
```

Comment:
This change doesn't seem related. May need to run `yarn install` again to get updated linting rules.